pkgs/applications/editors/spacevim/default.nix

```diff
@@ -0,0 +1,64 @@
+{ ripgrep, gitAndTools, fzf, makeWrapper, vim_configurable, vimPlugins, fetchFromGitHub, writeTextDir
+, stdenv, runCommandNoCC, remarshal, spacevim_config ? import ./init.nix }:
+with stdenv;
+let
+  # Once https://github.com/NixOS/nixpkgs/pull/75584 is merged we can use the TOML generator
+  toTOML = name: value: runCommandNoCC name {
+    nativeBuildInputs = [ remarshal ];
+    value = builtins.toJSON value;
+    passAsFile = [ "value" ];
+  } ''
+    mkdir $out
+    json2toml "$valuePath" "$out/${name}"
+  '';
```

Comment:
@Ma27 This generates the TOML from the Nix attrset now.
I default the value in the function definition above.
`spacevim_config ? import ./init.nix`